docs-next/pages/guides/document-fields.mdx

```diff
@@ -490,6 +490,8 @@ export default ({ children }) => {
       <DocumentFeaturesProvider>
         <div className="prose">
           <H1>How To Use Document Fields</H1>
+          <p>The [`document`](../apis/fields#document) field type provides a highly configurable rich content field based on [Slate](https://docs.slatejs.org/).
```

Comment:
@mitchellhamilton I'd love to include an introductory paragraph like this (exact wording TBD) before dropping people into the editor example with no other context. Is there an easy/correct way do that with markdown content, or would have we have write raw react/html for this? Links and formatting quickly becomes a bit fraught at that point.

Reply:
Without https://github.com/keystonejs/keystone/blob/5691466a6b30bf59d0a3f8d0a0fd0d4bd94c6210/docs-next/lib/prose-lite.ts#L1 being completely removed and replaced with using the MDXProvider, the only other option would be to have another mdx file for this bit of content and then import it and render it here.

Reply:
Importing from another mdx file sounds less gross than writing raw react, let me give that a shot and see what it looks like.

Reply:
Yep, that works pretty cleanly, thanks 👍